CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 # Changelog
 
-## 7.0.0 [#139](https://github.com/openfisca/country-template/pull/139)
+### [#144](https://github.com/openfisca/country-template/pull/144)
```

Comment:
Missing a version number, or should be integrated in the `7.0.0` release (preferred, since 7.0.0 was not released anyway).

I'm personally in favour of simply dropping that change altogether from the changelog, as it does not impact reusers.

Reply:
Sorry, didn't get that!